pkg/logstash.sysv

```diff
@@ -114,6 +114,27 @@ force_stop() {
   fi
 }
 
+config_test() {
+  # Check if a config file exists
+  if [ ! "$(ls -A $LS_CONF_DIR/*.conf 2> /dev/null)" ]; then
+    log_failure_msg "There aren't any configuration files in $LS_CONF_DIR"
```

Comment:
We could let it deal with it but it's less than 1 sec check vs a few seconds for logstash to start and notice there are not files.